kbase.yml

```diff
@@ -10,7 +10,7 @@ service-language:
     perl
 
 module-version:
-    0.0.15
+    0.0.16
 
 owners:
     [chenry,scanon,olson,jjeffryes,umaganapathyswork]
```

Comment:
@landml Can you add yourself to the authors so that you have the power to promote/register things.